core/src/main/codegen/templates/Parser.jj

```diff
@@ -4528,17 +4528,50 @@ SqlLiteral DateTimeLiteral() :
         return SqlParserUtil.parseTimestampLiteral(p, s.end(this));
     }
 |
-    <DATE> { s = span(); } <QUOTED_STRING> {
-        return SqlParserUtil.parseDateLiteral(token.image, s.end(this));
-    }
+    <DATE> { s = span(); } ((<QUOTED_STRING> | <BIG_QUERY_QUOTED_STRING>)
```

Comment:
Can you use the same indentation for '{' and '|' as other code.

Reply:
Addressed based on the rule added.